models/task/task.go

```diff
@@ -37,10 +38,10 @@ func FindTask(listID int64, taskID int64) (*TaskStruct, error) {
 	table := interfaceDB.Init()
 	defer table.Close()
 
-	var id, listId, userId int64
+	var id, listId, userId, projectId int64
```

Comment:
__[golint]__
* [golint] var listId should be listID
* [golint] var userId should be userID
* [golint] var projectId should be projectID